cognite/client/data_classes/contextualization.py

```diff
@@ -0,0 +1,231 @@
+import time
```

Comment:
nit: would name this file entity_matching - as we generally have the same names in _api and data_classes

Reply:
this is kind of pre-empting moving the pnid parser and such

Reply:
but for other resources we do not keep everything in the umbrella group in a single file, that would be like having data sets, labels and relationships in a single file called data_organization.py.

Its not a blocker for merging, but i'm not convinced it makes the most sense